docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
-This section explains the elements of the Trace View.
+Grafana's trace view provides an overview of a request as it travels through your system. The following sections provide detail on various elements of the trace view.
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view.png" class="docs-image--no-shadow" max-width= "900px" caption="Trace view" >}}
 
 ### Header
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-header.png" class="docs-image--no-shadow" max-width= "750px" caption="Screenshot of the trace view header" >}}
+The trace view header includes the following:
+
+- **Header title -** Shows the name of the root span and trace ID.
+- **Search -** Highlights spans containing the searched text.
+- **Metadata -** Various metadata about the trace.
 
-- Header title: Shows the name of the root span and trace ID.
-- Search: Highlights spans containing the searched text.
-- Metadata: Various metadata about the trace.
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-header.png" class="docs-image--no-shadow" max-width= "750px" caption="Trace view header" >}}
 
 ### Minimap
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-minimap.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view minimap" >}}
+**Minimap**m displays a condensed view of the trace timeline. Drag your mouse over the minimap to zoom into a smaller time range. This also updates the main timeline, making it easier to view shorter spans
+When zoomed in, hovering over the minimap displays the **Reset selectio** button, which resets the zoom.
 
-Shows condensed view or the trace timeline. Drag your mouse over the minimap to zoom into smaller time range. Zooming will also update the main timeline, so it is easy to see shorter spans. Hovering over the minimap, when zoomed, will show Reset Selection button which resets the zoom.
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-minimap.png" class="docs-image--no-shadow" max-width= "900px" caption="Trace view minimap example" >}}
 
-### Span filters
-
-![Screenshot of span filtering](/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png)
+### Timeline
 
-Using span filters, you can filter your spans in the trace timeline viewer. The more filters you add, the more specific are the filtered spans.
+Timeline shows list of spans within the trace. Each span row consists of the following components:
 
-You can add one or more of the following filters:
+- **Expand children -** Expands or collapses all the children spans of the selected span.
+- **Service name -** Name of the service logged the span.
+- **Operation name -** Name of the operation that this span represents.
+- **Span duration bar -** Visual representation of the operation duration within the trace.
```

Comment:
This might also be a good place to use a term and definition list instead of a bulleted list. 
```suggestion
- **Expand children** - Expands or collapses all the children spans of the selected span.
- **Service name** - Name of the service logged the span.
- **Operation name** - Name of the operation that this span represents.
- **Span duration bar** - Visual representation of the operation duration within the trace.
```